docker-compose.yaml

```diff
@@ -31,6 +31,7 @@ services:
       POSTGRES_PASSWORD: ${POSTGRES_SCHEMA_PASSWORD:-ppp}
       OVERNIGHT_JOB_CRONTAB: ${OVERNIGHT_JOB_CRONTAB:-5 4 * * *}
       OVERNIGHT_EXPORT_BATCH_SIZE: ${OVERNIGHT_EXPORT_BATCH_SIZE:-2000}
+      PARANOID_RETENTION_PERIOD: ${PARANOID_RETENTION_PERIOD:-30}
 
```

Comment:
You will need to add this setting to helm/values.yaml otherwise any overrides won't get picked up

Reply:
thanks, wasn't sure how to do this.